_includes/ome2021_program_agenda.html

```diff
@@ -10,7 +10,8 @@ <h2>{{ event.name }}</h2>
     <h5>Please Note</h5>
     <p>
         This schedule is subject to change and will be updated periodically.<br/>
-        The schedule will include pre-recorded videos organised by topic followed by group discussions.
+        The schedule will include pre-recorded videos organised by topic followed by group discussions.<br/>
+        All times are in UK - British Summer Time (GMT+1).
```

Comment:
I think I had moved this sentence under the `Schedule` section (as this is the only one which indicates day times)

Reply:
I missed it